config/core/test-configs.yaml

```diff
@@ -2481,12 +2499,14 @@ test_configs:
   - device_type: rk3328-rock64
     test_plans:
       - baseline
+      - kselftest-alsa
```

Comment:
Please also add baseline-nfs since kselftest uses NFS. This helps with identifying issues that are related to booting with NFS as opposed to kselftest.

@montjoie This platform is only available in BayLibre's lab so please confirm it's OK to enable these tests on it.

Reply:
@montjoie @broonie No news and no results:
https://staging.kernelci.org/test/job/kernelci/branch/staging-mainline/kernel/staging-mainline-20220415.0/plan/kselftest-alsa/

So I suggest we drop this now.